lib/audited/audit.rb

```diff
@@ -77,6 +75,21 @@ def associated
       associated_type&.constantize&.find_by(id: associated_id || associated_uuid)
     end
 
+    def user
```

Comment:
I want to make sure that no breaking changes are expected by changing `user` from an association to a method.

Reply:
This has been tested 👍